.github/workflows/autobahn.yml

```diff
@@ -14,7 +14,6 @@ on:
       - 'test/autobahn/**'
 permissions:
   contents: read
-  pull-requests: write
```

Comment:
why does this fix the nightly workflow?

I think we should just skip autobahn for nightly instead.

Reply:
We needed pull-requests write permission when we created a comment in the PR. This was unreliable. So we removed the comment creation but did not remove the permission. Now in a scheduled run, where we call this workflow as reusable workflow, it can not have more perissions than the calling workflow.

We should not skip the autobahn tests. If we had them in the first place, we would have seen the issues with the regressions in node core regarding buffers earlier.